client/components/mma/shared/benefits/BenefitsConfiguration.ts

```diff
@@ -37,7 +37,7 @@ export const benefitsConfiguration: {
 			isUnavailable: true,
 		},
 	],
-	supporterplus: [newsApp, supporterNewsletter, uninterruptedReading, adFree],
+	supporterplus: [newsApp, adFree],
```

Comment:
are they only supposed to be hidden on switch journey? this change would remove it from account overview too

Reply:
Good point - I think it's only for the switch journey